app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -544,6 +546,7 @@ class PinPasswordActivityTest {
 
   @Test
   fun testPinPassword_withUser_forgot_inputAdminPinAndNewPin_opensHomeActivity() {
+    TestPlatformParameterModule.forceEnableMultipleClassrooms(false)
```

Comment:
While tehese tests are okay, is there any reason why the  corresponding `opensClassroomListActivity` tests are restricted to run on Espresso?

Reply:
Not sure, why those were added. I have removed the restrictions.